lib/LedgerSMB/Locale.pm

```diff
@@ -85,8 +91,11 @@ package LedgerSMB::Locale;
 
 use strict;
 use warnings;
+use Exporter;
+
+use base qw( Locale::Maketext Exporter );
```

Comment:
Since `Locale::Maketext` didn't need to be `use`-d, I don't think Exporter needs to (or, for consistency: should) either.